de/manual/ch_Tools_Assistants.xml

```diff
@@ -49,1074 +49,801 @@
+        suchen</guilabel> zu nutzen, verwenden Sie im Menü den Eintrag &mc.ed.find;.
       </para>
 
-      <note>
-        <para>The first row of the <guilabel>Find Transaction</guilabel> dialog indicates that you are performing
-          a <guilabel>Split Search</guilabel>.
-        </para>
-      </note>
+      <para>Die Suchergebnisse hängen von der Registerkarte ab, von der aus Sie die Suche starten und ob das
+        gewählte Datenfeld zur Buchung oder einem Buchungsteil gehört.
+        <itemizedlist>
+          <para>Diese Felder gehören zur Buchung:
+          </para>
 
-      <para>There are two panes in the <guilabel>Find Transaction</guilabel> dialog. The top pane contains the
-        <guilabel>Search Criteria</guilabel> buttons and the bottom pane contains the <guilabel>Type
-        of Search</guilabel> selection.
-      </para>
+          <listitem>
+            <para>Beschreibung
+            </para>
+          </listitem>
 
-      <para>There are two buttons in the top of the <guilabel>Search Criteria</guilabel> pane. The left button
-        allows you to <guibutton>Add</guibutton> another criteria row to search for multiple
-        criteria. The existing criteria row needs to be completed before adding a new row. The right
-        button is used to <guilabel>Search for items where all criteria are met</guilabel> or
-        <guilabel>Search for items where any criteria are met</guilabel>. The <guilabel>all criteria
-        are met</guilabel> search requires all of the criteria rows to have at least one match. The
-        <guilabel>any criteria are met</guilabel> search requires only one of the criteria rows to
-        be matched.
-      </para>
+          <listitem>
+            <para>Bemerkung
+            </para>
+          </listitem>
 
-      <para>The criteria row in the <guilabel>Search Criteria</guilabel> pane is used to combine different
-        criteria buttons (see table <xref linkend="tool-find-bttns" /> for a comprehensive list of
-        combinations)
-      </para>
+          <listitem>
+            <para>Nummer
+            </para>
+          </listitem>
 
-      <para>In the <guilabel>Type of Search</guilabel> pane there are five selections. The first option is
-        always enabled, the following three are available when you invoke the &gmi.ed.find; option
-        from a register window or after performing the first <guibutton>Find</guibutton> if the
-        &gmi.ed.find; command was invoked from any other window. The last option is available only
-        for <link linkend="tool-find-bsnss">Business search</link>.
-      </para>
+          <listitem>
+            <para>Buchungsdatum
+            </para>
+          </listitem>
+        </itemizedlist>
 
-      <variablelist>
-        <varlistentry>
-          <term><guilabel>New Search</guilabel></term>
+        <itemizedlist>
+          <para>Wohingegen die anderen Felder in den Buchungsteile enthalten sind und somit jeweils einem einzigen
```

Comment:
Wohingegen die anderen Felder in den `<emphasis>`Buchungsteile<b>n</b>`</emphasis`>